libs/sdk-ui-dashboard/src/model/commands/dashboard.ts

```diff
@@ -2,69 +2,165 @@
 
 import { DashboardConfig } from "../types/commonTypes";
 import { IWorkspacePermissions } from "@gooddata/sdk-backend-spi";
+import { IDashboardCommand } from "./base";
 
 /**
- * All available command types.
+ * Loads dashboard from analytical backend.
+ *
+ * @internal
+ */
+export interface LoadDashboard extends IDashboardCommand {
+    readonly type: "GDC.DASHBOARD.CMD.LOAD";
+    readonly payload: {
+        readonly config?: DashboardConfig;
+        readonly permissions?: IWorkspacePermissions;
+    };
+}
+
+/**
+ * Creates the LoadDashboard command. Dispatching this command will result in the load of all
+ * the essential data from the backend and initializing the state of Dashboard to a point where the
+ * dashboard can be rendered.
  *
+ * @param config - configuration to use for for the Dashboard; you MAY NOT provide any configuration or MAY provide
+ *  partial configuration. During the LoadDashboard processing the Dashboard component WILL resolve all the missing
+ *  parts by reading them from the backend
+ * @param permissions - permissions to use when determining whether the user is eligible for some actions with the
+ *  dashboard; you MAY NOT provide any permissions (undefined) and the Dashboard component WILL load the permissions
```

Comment:
Nitpick: you MAY NOT souds like it is forbidden, which is not the case right? This is also used some lines above as well.

Reply:
reworded